subsys/kafka/src/main/java/org/commonjava/indy/subsys/kafka/conf/LogKafkaConfig.java

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Are we still using this for logging? If not, maybe we should mark this as deprecated for future deletion.